docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
+    environment:
+      - FF_APP_ENV=production
+    volumes:
+      - ./storage:/var/www/firefly-iii/storage
```

Comment:
When I used a volume mount, I got an error that the sessions folder did not exist in storage/framework as soon as I logged in. However, if you know a way to get it working with a Docker volume, that would also be fine. I think the volume is mounted after the files are copied (i.e. copying is a build step; volume mounting is done when running the container). Therefore, the folder might be empty when the application starts with a Docker volume.